### PR TITLE
Add Swiss Federal Office for Customs as a forex source

### PR DESCRIPTION
**Pull request by yotamhc**

### CODE REVIEW

**Review by dfinity-ryancroote — CHANGES_REQUESTED** [review]
One suggestion.

This PR also contains Italy and Georgia. We will need to merge your prior PR first in order to settle this one.

File: src/xrc/src/forex/switzerland.rs
Comment (yotamhc) [reply src/xrc/src/forex/switzerland.rs]: Thanks, I applied your suggestion (just fixed the second `if`)

**Comment by yotamhc** [comment]
I will merge this PR with `main` once #208 is merged, conflict resolution should be easy.

**Review by dfinity-ryancroote — APPROVED** [review]

**Pull request merged**
